docs/release_guide.md

```diff
@@ -152,6 +152,8 @@ On the docs site's main branch, edit the file [https://github.com/openservicemes
 1. Copy the table and replace the existing table in the file [https://github.com/openservicemesh/osm-docs/blob/main/content/docs/guides/troubleshooting/control_plane_error_codes.md](https://github.com/openservicemesh/osm-docs/blob/main/content/docs/guides/troubleshooting/control_plane_error_codes.md).
 1. If there were updates to the table, make a PR in the OSM docs repository.
 
+### 5. Update uninstall documentation
+If any changes were made to the [OSM cleanup hook](https://github.com/openservicemesh/osm/blob/main/charts/osm/templates/cleanup-hook.yaml), make a PR in the OSM docs repository updating the list of resources deleted during uninstall in the [uninstall guide](https://github.com/openservicemesh/osm-docs/blob/main/content/docs/guides/uninstall.md#uninstall-osm-control-plane-and-remove-user-provided-resources) and the commands in the [uninstall troubleshooting](https://github.com/openservicemesh/osm-docs/blob/main/content/docs/guides/troubleshooting/uninstall.md) manual deletion instructions.
```

Comment:
I feel as though this seems out of place in the release guide. This should be a part of the PR lifecycle so that it is detected much earlier during code review, vs adding to process overhead during the release workflow.

I'd rather have a workflow as follows:
1. Someone updates the cleanup hook
2. Reviewer detects doc changes required
3. Opens an issue in osm-docs to track this and makes it a release blocker if applicable

Reply:
I didn't see this before posting my comment, but mine says basically the same thing.